codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/AsyncClientClass.java

```diff
@@ -229,7 +229,14 @@ private MethodSpec constructor(TypeSpec.Builder classBuilder) {
                         .addStatement("this.clientHandler = new $T(clientConfiguration)", AwsAsyncClientHandler.class)
                         .addStatement("this.clientConfiguration = clientConfiguration.toBuilder()"
                                       + ".option($T.SDK_CLIENT, this)"
-                                      + ".build()", SdkClientOption.class);
+                                      + ".option($T.API_METADATA, $S + \"#\" + $T.VERSION)"
+                                      + ".build()",
+                                      SdkClientOption.class,
+                                      SdkClientOption.class,
```

Comment:
Duplicate `SdkClientOption.class`

Reply:
Why is this a concern?

Reply:
Oh I missed `.option(SdkClientOption.SDK_CLIENT, this)` and thought we only needed one. nvm then!